modules/ducktests/pom.xml

```diff
@@ -188,6 +193,23 @@
                             <excludeTransitive>false</excludeTransitive>
                         </configuration>
                     </execution>
+                    <execution>
```

Comment:
is it possible to make it as we do for jackson dependencies? Check specs implementation, `__jackson` method

Reply:
fixed